install.sh

```diff
@@ -485,14 +485,22 @@ maybe_enable_userns_sysctl() {
     return
   fi
 
-  if [ "$(sysctl -n kernel.unprivileged_userns_clone)" = "1" ]; then
+  local OLD_VALUE="$(< /proc/sys/kernel/unprivileged_userns_clone)"
+
+  if [ "$OLD_VALUE" = "1" ]; then
     # Already enabled.
     return
   fi
 
   # Enable it.
   echo "NOTE: Enabling unprivileged user namespaces because you passed -d."
-  sysctl -wq kernel.unprivileged_userns_clone=1
+  if ! sysctl -wq kernel.unprivileged_userns_clone=1; then
```

Comment:
For some reason I assumed Docker installs required userns, but in retrospect that doesn't make sense. OK, removed the warning.